examples/clients/python/seafowl/__main__.py

```diff
@@ -40,10 +42,13 @@
         sql = "; ".join(f.readlines())
 else:
     sql = " ".join(args.rest)  # type:ignore
-
-status_code, data = query(
-    SeafowlConnectionParams(endpoint, password, None), sql, append_url_suffix=False
-)
-print(f"code: {status_code}")
-if data:
-    print(json.dumps(data, indent=4))
+try:
+  status_code, data = query(
+      SeafowlConnectionParams(endpoint, password, None), sql, append_url_suffix=False
+  )
+  print(f"code: {status_code}")
+  if data:
+      print(json.dumps(data, indent=4))
+except HTTPError as error:
+    print(f"Error: {error.response.status_code}")
+    print(error.response.text)
```

Comment:
I'd print this to stderr and exit with a non-zero code so that anything downstream knows this crashed.

Reply:
That's a good idea, updating the code accordingly!